### ISSUE TITLE
Modifications for February 2024

### ISSUE THREAD

**Issue by ryanpcmcquen** [Tech-How/Light-Theme-for-YouTube-Music#4]
I added this to the bottom of the stylesheet, because the app drawer dark background was persisting:

```css
#contentContainer.tp-yt-app-drawer,
.style-scope.ytmusic-app {
    background-color: #F4F4F4;
}
```

**Comment by Tech-How** [comment]
Hi, thanks for your feedback. I've been meaning to update this for a while but have been reluctant to find the time to. Will definitely implement this in the next release!

**Comment by Tech-How** [comment]
Fixed in version 2.0 :)
